lua/octo/pickers/snacks/provider.lua

```diff
@@ -638,6 +689,33 @@ function M.search(opts)
       final_keys[cfg.picker_config.mappings.copy_url.lhs] = { "copy_url", mode = default_mode }
     end
 
+    if not custom_actions_defined["copy_sha"] then
+      final_actions["copy_sha"] = function(_picker, item)
+        if item.kind == "pull_request" then
+          -- For PR search results, we need to fetch the PR details to get the head SHA
+          utils.info "Fetching PR details for SHA..."
+          local owner, repo = item.repository.nameWithOwner:match "([^/]+)/(.+)"
```

Comment:
utils.split_repo or something similar should perform this split

Reply:
Done in b61d2cf0b0b684c576fb533e0999c6a1d40c6a55